test/test18.sh

```diff
@@ -0,0 +1,9 @@
+printf "build/marc --output=test.tmp input/test18.opl\n";
```

Comment:
Should be .. 

`printf "build/alex --output=tmp/test18.lex input/test18.opl\n";`